### ISSUE TITLE
Show author picture near name

### ISSUE THREAD

**Issue by aldrinjenson** [IEDCMEC/iedc-blog#23]
![image](https://user-images.githubusercontent.com/53407417/188055293-4da9bd0c-56ef-4753-b2df-fbaf8ada2260.png)

Similar to how in Medium.com we get a round profile picture of the blog author, it'd be great if could show the profile picture of the authors near to their names. 

![image](https://user-images.githubusercontent.com/53407417/188055451-79feeed6-d875-4948-ab74-ba0a973faa58.png)
 
You can collect the image as a url from the metadata of a post similar to how the author name and description is being collected.
Refer to [this](https://github.com/IEDCMEC/iedc-blog/pull/19) PR for more.
Basically declare a new property imgUrl and when an image link is present, show the image. else show a sample default image to make the spacings consistent. Refer Medium as example. 
You'd have to modify the following files mostly:
https://github.com/IEDCMEC/iedc-blog/blob/main/themes/anatole/layouts/partials/author.html
https://github.com/IEDCMEC/iedc-blog/blob/main/assets/css/author.css

@IEDCMEC/blog-team

**Comment by jaison080** [comment]
The issue has been resolved and the PR has been put up.Do check it and merge...

**Comment by amaldevcd** [comment]
Closed by pr #27
